packages/frontend/src/components/UserSettings/GithubSettingsPanel/index.tsx

```diff
@@ -82,7 +89,34 @@ const GithubSettingsPanel: FC = () => {
     const { data, isError, isInitialLoading } = useGitHubRepositories();
     const deleteGithubInstallationMutation =
         useDeleteGithubInstallationMutation();
+
+    const status = useSearchParams('status');
+    const { showToastWarning } = useToaster();
+    const isWaitingForGithubRequest = status === 'github_request_sent';
```

Comment:
does the search param clear after installation?

Reply:
nope, but it is ignored if the request to github returns the right results